modules/services/karabiner-dk/default.nix

```
@@ -0,0 +1,55 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  cfg = config.services.karabiner-dk;
+
+  parentAppDir = "/Applications/Nix Apps";
+in
+{
+  meta.maintainers = [ lib.maintainers.auscyber or "auscyber" ];
+  options.services.karabiner-dk = {
+    enable = mkEnableOption "Karabiner-DK";
+    package = mkPackageOption pkgs "karabiner-dk" { };
+  };
+  config = mkIf cfg.enable {
+    environment.systemPackages = [
+      cfg.package
+    ];
+    environment.extraAppPaths = [
+      "/Library/Application Support/org.pqrs/Karabiner-DriverKit-VirtualHIDDevice"
+    ];
+    launchd.daemons.Karabiner-DriverKit-VirtualHIDDevice-Daemon = {
+      serviceConfig.ProgramArguments = [
+        "/Library/Application Support/org.pqrs/Karabiner-DriverKit-VirtualHIDDevice/Applications/Karabiner-VirtualHIDDevice-Daemon.app/Contents/MacOS/Karabiner-VirtualHIDDevice-Daemon"
+      ];
```

You don't have to copy anything into `/Library` to run here
```suggestion
      command = "${cfg.package}/Library/Application\\ Support/org.pqrs/Karabiner-DriverKit-VirtualHIDDevice/Applications/Karabiner-VirtualHIDDevice-Daemon.app/Contents/MacOS/Karabiner-VirtualHIDDevice-Daemon";
```